plugins/lookup/lookup.py

```diff
@@ -295,22 +299,40 @@ def run(self, terms, variables=None, **kwargs) -> list[str]:
         )
 
     @staticmethod
-    def get_urls(base_url: str, api_url: str, identity_url: str) -> tuple[str, str]:
-        if base_url:
-            base_url = base_url.rstrip("/")
+    def sanitize_urls(*args):
+        sanitized_inputs = []
+        for input_var in args:
+            input_var = str(input_var).strip()
+            input_var = input_var.rstrip("/")
+            sanitized_inputs.append(input_var)
+        return sanitized_inputs
+
+    @staticmethod
+    def get_urls(
+        base_url: str = None, api_url: str = None, identity_url: str = None
+    ) -> tuple[str, str]:
+        if base_url != BITWARDEN_BASE_URL:
             api_url = f"{base_url}/api"
             identity_url = f"{base_url}/identity"
-        elif api_url and identity_url:
-            return api_url, identity_url
-        elif not base_url and not api_url and not identity_url:
-            api_url = BITWARDEN_API_URL
-            identity_url = BITWARDEN_IDENTITY_URL
+            return (api_url, identity_url)
         else:
-            display.error(API_IDENTITY_URL_ERROR.format(api_url, identity_url))
-            raise AnsibleError(
-                API_IDENTITY_URL_ERROR.format(api_url, identity_url)
-            ) from None
-        return api_url, identity_url
+            if api_url == BITWARDEN_API_URL and identity_url == BITWARDEN_IDENTITY_URL:
+                return (api_url, identity_url)
+            elif (
+                api_url != BITWARDEN_API_URL and identity_url == BITWARDEN_IDENTITY_URL
+            ) or (
+                api_url == BITWARDEN_API_URL and identity_url != BITWARDEN_IDENTITY_URL
+            ):
```

Comment:
I think we do not need first condition at all, since we already have the return.

```suggestion
            if (
                api_url != BITWARDEN_API_URL and identity_url == BITWARDEN_IDENTITY_URL
            ) or (
                api_url == BITWARDEN_API_URL and identity_url != BITWARDEN_IDENTITY_URL
            ):
```